docs/advanced/transform-content.md

```diff
@@ -0,0 +1,40 @@
+# Transform Content
+
+As we previously stated, you will most likely use MDX through a loader or framework, but it is also possible to interact with or manipulate raw MDX content.  Since MDX is a part of the [unified][unified] ecosystem, there are many utilities we can use to work with MDX.
```

Comment:
```suggestion
With MDX you can interact with or manipulate raw MDX content.  Since MDX is a part of the [unified][unified] ecosystem, there are many utilities you can use to work with MDX.
```